patches/net/minecraft/world/damagesource/DamageEffects.java.patch

```diff
@@ -26,6 +26,20 @@
 +        this(p_270875_, () -> p_270383_);
 +    }
 +
++    /**
++     * Creates a new DamageEffects with the specified ID and sound. Store the created enum in a static final field.<br>
++     * Use an enumextender.json to link to your {@link net.neoforged.fml.common.asm.enumextension.EnumProxy} in order to create new entries for this enum.<br>
++     * Example usage:
```

Comment:
Use paragraph tags rather than line break tags.

Reply:
this aint no approval! : screm :

Fixed